docs/@aptos-labs/ts-sdk-0.0.8/classes/TransactionWorker.html

```diff
@@ -1,6 +1,6 @@
 </div></section><section class="tsd-index-section"><h3 class="tsd-index-heading">Properties</h3><div class="tsd-index-list"><a href="TransactionWorker.html#account" class="tsd-index-link"><svg class="tsd-kind-icon" viewBox="0 0 24 24"><use href="#icon-1024"></use></svg><span>account</span></a>
-<a href="TransactionWorker.html#accountSequnceNumber" class="tsd-index-link"><svg class="tsd-kind-icon" viewBox="0 0 24 24"><use href="#icon-1024"></use></svg><span>account<wbr/>Sequnce<wbr/>Number</span></a>
+<a href="TransactionWorker.html#accountSequenceNumber" class="tsd-index-link"><svg class="tsd-kind-icon" viewBox="0 0 24 24"><use href="#icon-1024"></use></svg><span>account<wbr/>Sequence<wbr/>Number</span></a>
```

Comment:
Please don't change this in the documentation.

Unfortunately, looks like the name is misspelled on the actual TransactionWorker, and that will need to be updated in the next release.  At that point the docs will be regenerated correctly.

My suggestion here would be to leave all the existing docs alone, fix any other spelling mistakes, and then with the next release, we'll update the autogenerated documentation for the next release.